enclave/random.c

```diff
@@ -82,9 +82,11 @@ oe_result_t oe_random_internal(void* data, size_t size)
 
     /* Generate random data (synchronize access to _drbg instance) */
     rc = mbedtls_ctr_drbg_random(&_drbg, data, size);
-
     if (rc != 0)
+    {
+        OE_TRACE_ERROR("rc = 0x%x\n", rc);
```

Comment:
I don't know if this should be logged. 
Random is used for many cryptographic operations. Would logging a failed random open up side-channels?

Reply:
I am not sure.
MBEDTLS_ERR_CTR_DRBG_ENTROPY_SOURCE_FAILED and 
MBEDTLS_ERR_CTR_DRBG_REQUEST_TOO_BIG are the only possible error codes. 
As long as we don;t  reveal the data and size in this error message, it appears fine to me.